Tests/LibWeb/Layout/input/block-and-inline/block-inside-inline.html

```diff
@@ -0,0 +1,6 @@
+foo
```

Comment:
Please add `<!DOCTYPE html>` as the first line and add a newline at the end of this file.